pkg/services/mssql/catalog.go

```diff
@@ -518,6 +542,7 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 						"1200 DWUs",
 					},
 				},
+				ProvisionParamsSchema: GetDBaseProvisionParametersSchema(),
```

Comment:
Looking at this particular catalog raised a yellow flag for me...

I do realize the OSB spec allows schemas to be defined plan-by-plan and we have to play ball with that, _but_ we've made an architectural decision that all plans of a given service vary only in QoS, and thus have common sets of parameters.

Would it make more sense, for us and for module implementors if we allow the schemas to be added to the `ServiceProperties` instead of the `PlanProperties` and then we just deal with "spraying" those schemas from the services down to their constituent plans behind the scenes?

Reply:
Yes, I think it would. Implemented as-is because of the broker spec, but I think it would simplify things.